src/mesh/gmsh_reader.cpp

```diff
@@ -55,8 +54,12 @@ assign_1d_boundary_ids( const std::map<unsigned int, dealii::types::boundary_id>
 }
 
 template <int dim>
-void rotate_indices(std::vector<unsigned int> &numbers, const unsigned int n_indices_per_direction, const char direction)
+void rotate_indices(std::vector<unsigned int> &numbers, const unsigned int n_indices_per_direction, const char direction, const bool /*mesh_reader_verbose_output*/)
```

Comment:
Last argument is used for debug, and should be kept in case there is an error when reading the grid. I added some debug statements in the latest commit.